### DISCUSSION TITLE
Pastelería VUE

### DISCUSSION THREAD

**Discussion by ValencianoAlan** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#524] [Q&A] [answered]
Hola chicos, les quería pedir ayuda con respecto a esta práctica, alguien tiene alguna idea de cómo hacer la página de la pastelería con vue? No encuentro como iniciar 🥺

**Comment by doguedogue** [comment, marked as answer]
Te recomendaria ver algunos tutoriales para aprender más sobre el framework.

Yo he visto dos o tres tutoriales y me han servido, checate a Fernado Herrera https://www.youtube.com/watch?v=kY9ljqqw80k
Tiene cursos gratis o de paga en Udemy.

1. Pasar tu página de Pasteleria a vue, cada elemento creas un componente (header, footer, navbar, etc)
2. Usar el router para cambiar de vistas (presentación, pedidos, etc.)
3. Usar los estados para pasar la info de una página a otra
4. NO TIENES QUE GUARDARLA (aunque se podría, pero se requiere el backend)

Vamos tu puedes, a estudiar y luego a programar 🚀🚀🚀

**Reply by ValencianoAlan** [reply]
muchas gracias, me acostumbre a entrar directo al codigo sin antes investigar, ya con el video y otros tutoriales me pueden servir
